docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -61,20 +61,14 @@ new CA using the following command:
 <ScopedBlock scope={["oss", "enterprise"]}>
 
 ```code
-# Log in to your cluster with tsh so you can use tctl from your local machine.
-# You can also run tctl on your Auth Service host without running "tsh login"
-# first.
-$ tsh login --proxy=teleport.example.com --user=myuser
-$ tctl auth export --type=windows >user-ca.cer
+$ curl 'https://teleport.example.com/webapi/auth/export?type=windows' > user-ca.cer
```

Comment:
If we use the `Var` component instead of the example Teleport address (so a user inputs their own Teleport cluster address) we don't need to use `ScopedBlock`s.

Reply:
Applied `<Var name="proxy"/>` to every usage of the command :+1: